src/pages/Organizations/organizationInventory.js

```diff
@@ -28,15 +31,52 @@ const AddItemButton = getMyAgent(({agent, loading, error}) => {
     } else {
         for (let i = 0; i < agent.agentRelationships.length; i++) {
             if(agent.agentRelationships[i].object.id === orgId){
-                return (<Button className="ui right floated primary" onClick={createItem}>Add Item</Button>);
+                connected = true;
             }
-
         }
         return <div/>
     }
 
 });
 
+class AddItemButton extends React.Component {
+
+    createItem = () => {
+        history.push("/CreateInventoryItem/"+orgId);
```

Comment:
Try merging dev down into here.  We will want the modal changes in here before we merge I believe